sonic-thermalctld/scripts/thermalctld

```diff
@@ -640,6 +640,36 @@ class TemperatureUpdater(logger.Logger):
                                         )
         warning = warning | temperature_status.under_temperature
 
+        THRESH_NAME_LIST = ['high_threshold', 'low_threshold', 'critical_high_threshold', 'critical_low_threshold']
+        THRESH_COMP_VAL_LIST = [high_threshold, low_threshold, high_critical_threshold, low_critical_threshold]
+        THRESH_FUNC_LIST = [thermal.set_high_threshold, thermal.set_low_threshold, thermal.set_high_critical_threshold, thermal.set_low_critical_threshold]
+
+        ovalues = self.table.get(name)
+
+        for first, second in ovalues[1]:
+            if second == NOT_AVAILABLE:
+                continue
+
+            th_type = 0
+            if first in THRESH_NAME_LIST:
+                th_type = THRESH_NAME_LIST.index(first)
+            else:
+                continue
+
+            if second != str(THRESH_COMP_VAL_LIST[th_type]):
+                try:
+                    if THRESH_FUNC_LIST[th_type](float(second)) is True:
```

Comment:
Why you get a value from STATE_DB and set it to the platform?

Reply:
This was misunderstood by the SONiC DB architecture and now I have moved the new values to CONFIG_DB as you suggested.